images/vault/tests/main.tf

```diff
@@ -17,9 +17,11 @@ data "oci_string" "ref" {
   input    = each.value
 }
 
+resource "random_pet" "suffix" {}
+
 resource "helm_release" "vault" {
   name             = "vault"
-  namespace        = "vault-system"
+  namespace        = "vault-system-${random_pet.suffix.id}"
```

Comment:
Generally we randomize the helm install name too

Reply:
+1